client/src/styles/Home.css

```diff
@@ -1,10 +1,67 @@
 .home-header {
   background-color: #282c34;
-  min-height: 100vh;
+  min-height: 10vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: center;
-  font-size: calc(10px + 2vmin);
+  font-size: calc(10px + 0.5vmin);
   color: white;
 }
+.logo {
+  float: left;
```

Comment:
If you use Semantic UI most of the CSS here will not be needed.

Reply:
I have a problem in the setting the main image in the page , I tried to stretch it to be as the width of the page but it did not respond also I want to put it in the center of the page and also no response :(